src/ai/backend/agent/docker/agent.py

```diff
@@ -785,6 +785,21 @@ def _mount(type: MountTypes,
                         os.chown(ssh_dir / 'authorized_keys', uid, gid)
                         os.chown(work_dir / 'id_container', uid, gid)
 
+        for dotfile in internal_data['dotfiles']:
```

Comment:
Please make it backward-compatible when possible:

```suggestion
        for dotfile in internal_data.get('dotfiles', []):
```